waspc/README.md

```diff
@@ -34,7 +34,32 @@ It can then also run that web app for you, deploy it (not yet but that is coming
 ### Setup
 We use the [`cabal`](https://cabal.readthedocs.io/) command line tool to build
 the project. The best way to install it is via
-[ghcup](https://www.haskell.org/ghcup/). The `ghcup tui` terminal user interface
+[ghcup](https://www.haskell.org/ghcup/) (if you're running an M1/M2 mac make sure to run it in rosetta).
```

Comment:
So what exactly do you have to run in rosetta -> `ghcup`, during installation, or `cabal` later when building the project, or both?

Reply:
You need to install `ghcup` and then I suppose also run `cabal` with rosetta (installing ghcup in m1 mode and then trying to run it or `cabal` didn't work)